_posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown

```diff
@@ -0,0 +1,49 @@
+How? Just fill out the Mentor Interest Form: [go.hackillinois.org/mentor](http://go.hackillinois.org/mentor)
+
+What are the commitments and responsibilities of a mentor?
+Mentors lead a group of students in addressing issues on projects that the mentors contribute to. Unlike at other collegiate hackathons, mentors don't just support students in fixing bugs here and there; mentors work together with students, guiding them through the open source workflow to make meaningful contributions.
+
+What sort of contributions have groups made in the past?
+In the past, HackIllinois has seen students make contributions to a projects in the areas of JavaScript, Android, Python, Linux, and more! Groups have opened PRs to projects such as PrairieLearn, a problem-driven learning system built at the University of Illinois; Puppeteer, Chrome's headless node API; and Rust, a systems programming language that's a favorite amongst developers. As a mentor, you can bring projects that you maintain or contribute to---anything you believe you can guide students through making contributions to.
+
+How many people will I be working with?
+Group sizes vary, but we find that the most successful groups have a ratio of 4-6 students per mentor and have 1-3 mentors that come ready with a handful features and fixes for the students to get started on.
+
+What are the skill levels of the students?
+Although there is a wide range of skill levels, most students are well-versed with writing code for classes or personal projects, but don't necessarily have familiarity with working in larger codebases or open source workflows. Mentors get to help students become familiar with the open source aspects of development!
+
+What do I get out of attending HackIllinois?
+Mentors get to inspire and teach students about open source, helping them to become active members in the community. As a mentor, you would work closely with students to help resolve issues on your projects. Mentors also get to meet leaders and other mentors at the event. Of course, we will also provide food and snacks throughout the weekend, as well as tons of swag!
+Team members will be able provide details regarding travel and accommodations based on your individual situation following initial conversations.
+
+What happens when I fill out this Mentor Interest Form?
+We will review submissions, and a team member will reach out to you via email to learn more about your projects and answer any questions you may have. The Mentor Interest Form is a way for you to get connected with our team and learn more; there is no commitment to attend the event based on form submission.
+
+I won't be able to make the dates this year, but I'd like to get updates about other events.
+Feel free to fill out the Mentor Interest Form anyway. We can contact you about other opportunities, such as our fall HackNights or HackIllinois in future years.
```

Comment:
potentially provide one more line here about HackNights to give more information